ADAL/src/workplacejoin/mac/ADWorkPlaceJoinUtil.m

```diff
@@ -22,14 +22,292 @@
 // THE SOFTWARE.
 
 #import "ADWorkPlaceJoinUtil.h"
+#import "ADKeychainUtil.h"
+#import "ADLogger+Internal.h"
+#import "ADWorkPlaceJoinConstants.h"
+#import "ADRegistrationInformation.h"
+
+// Convenience macro for checking keychain status codes while looking up the WPJ information.
+#define CHECK_KEYCHAIN_STATUS(_operation) \
```

Comment:
macro parameters should be in all caps to avoid confusion with any other variables that might exist.